en/flight_modes/follow_me.md

```diff
@@ -8,16 +8,22 @@ author: Jimmy Johnson
 
 *Follow Me* mode allows a multicopter to autonomously follow and track another system that is broadcasting its position using the [FOLLOW_TARGET](https://mavlink.io/en/messages/common.html#FOLLOW_TARGET) MAVLink message.
 
-The vehicle will automatically yaw to face and follow the target from a specified [relative position](#NAV_FT_FS), [horizontal separation](#NAV_FT_DST) and [height](#NAV_MIN_FT_HT) above the home position.
-By default it will follow from directly behind the target at a distance of 8 meters, and a height of 8 meters above the home/arming position.
-While in this mode no user input is required.
+![Follow-me Concept](../../assets/flight_modes/followme_concept.png)
 
-:::tip
-PX4 currently ignores the altitude of/from the target and follows at a constant altitude above home.
-This limitation is because altitude sources from GPS on ground stations are often inaccurate.
-:::
+The vehicle will automatically yaw to face and follow the target from a specified [relative angle](#NAV_FT_FS), [distance](#NAV_FT_DST) and [height](#NAV_FT_HT) and altitude, depending on the [altitude control mode](#NAV_FT_ALT_M)).
+
+By default it will follow from directly behind the target at a distance of 8 meters, and a height of 8 meters above the home(arming) position.

```

Comment:
```suggestion
By default it will follow from directly behind the target at a distance of 8 meters, and a height of 8 meters above the home (arming) position.

```